--- Dependencies.cmake.bck	2020-06-12 18:58:20.498901182 +0000
+++ Dependencies.cmake	2020-06-12 18:58:53.911083192 +0000
@@ -649,9 +649,11 @@
   caffe2_update_option(USE_FBGEMM ON)
 else()
   caffe2_update_option(USE_FBGEMM OFF)
-  message(WARNING
-    "Turning USE_FAKELOWP off as it depends on USE_FBGEMM.")
-  caffe2_update_option(USE_FAKELOWP OFF)
+  if(USE_FAKELOWP)
+    message(WARNING
+      "Turning USE_FAKELOWP off as it depends on USE_FBGEMM.")
+    caffe2_update_option(USE_FAKELOWP OFF)
+  endif()
 endif()
